test/test.js

```diff
@@ -0,0 +1,55 @@
+import test from 'ava';
+import fs from 'fs-extra';
+import webpack from 'webpack';
+import CSSStatsPlugin from '../index.js';
+import path from 'path';
+var pluginOptions = {
+    outputSuffix: "-CSSRaport",
+    outputPath: "./test/output",
```

Comment:
jeśli to `tmp` to lepiej użyć:

```js
import os from 'os';

const tmp = os.tmpDir(); // <- lub `os.tmpdir();, któryś zapis jest zdeprecjonowany
```

Dzięki temu w projekcie nie będzie trzeba mieć katalogu tymczasowego i rezultaty będą w systemowym katalogu tymczasowym